Modules/Bridge/NumPy/wrapping/PyVectorContainerBuffer.i.in

```diff
@@ -20,8 +20,8 @@
         else:
             shape   = (itksize,)
 
-        pixelType     = "@PixelType@"
-        numpydatatype = _get_numpy_pixelid(pixelType)
+        element_type     = "@ElementType@"
```

Comment:
`element_type` is already defined a few lines earlier.  Maybe add a comment somewhere near to indicate why the same variable name is taking on different values in different contexts.  Alternatively, rename one of the variables?

Reply:
Good point -- changing the above name to `container_element_type` for clarity.